src/Image.ts

```diff
@@ -706,6 +707,17 @@ export class Image {
   ): Image {
     return drawCircleOnImage(this, center, radius, options);
   }
+
+  // FEATURE MATCHING
+  /**
+   * Compute the intensity centroid of an image for each channel.
+   *
+   * @returns The intensity centroid of each channel of the image.
+   */
+  public getIntensityCentroid(): Point[] {
+    return getIntensityCentroid(this);
+  }
```

Comment:
I added this one, because I know that it is a commonly used method besides feature matching. Typically, our optics teacher told us it was used to enhance pictures of stars by realigning inidividual shots on their center of mass. I can still remove it if you don't find it relevant.

Here is an example where image centroid is useful in astronomy: "For the process of realignment, we take image centroid measurements in 3 × 3 pixel squares centered on the target, and convert them from pixel coordinates into distances from the middle of the 5 × 5 postage stamp."

https://iopscience.iop.org/article/10.3847/1538-4365/ac3a11#apjsac3a11s2-4

Reply:
The point is not the usefulness of the methods, it's to avoid having too many things at the top level. We will remove many others from the image class.